std_msgs/Header header
spomp/LocalReachability[] reachabilities
